tools/PSABootPackageCreator/PSABootPackageCreator/Crypto/PSACrypto.cs

```diff
@@ -55,7 +55,8 @@ public byte[] Encrypt(byte[] plainData)
 
```

Comment:
@ozdemirfadime @solakogullarielif how do we pass Private key? I could not see you are passing the key.

Reply:
we committed only last review so we will one more commit that contain a private key